resources/config_plugins.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Cookbook:: grafana
+# Resource:: config_plugins
+#
+# Copyright:: 2018, Sous Chefs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configures the [plugins] section of the grafana config file
+
+property  :instance_name, String, name_property: true
+property  :allow_loading_unsigned_plugins, String, default: ''
```

Comment:
I feel like this should be an array that we then join into a csv in code.

Reply:
Yeah i agree, we should take an array of names and join them in code. We should also update the example and tests to validate that the array was joined correctly